jetstream/bigquery_client.py

```diff
@@ -45,7 +45,7 @@ def add_labels_to_table(self, table_name: str, labels: Mapping[str, str]) -> Non
 
     def _current_timestamp_label(self) -> str:
         """Returns the current UTC timestamp as a valid BigQuery label."""
-        return str(int(time.mktime(datetime.utcnow().timetuple())))
+        return str(int(time.time()))
```

Comment:
`time()` returns the number of seconds since the epoch, which is what we want since this timestamp will get compared to the `committed_date` of commits which is also the number of seconds since the epoch.

[`mktime()`](https://docs.python.org/3/library/time.html#time.mktime) expresses the time in local time, not UTC. So I think we ran into problems here comparing UTC timestamps to local timestamps. In some cases it would have appeared as if there have been no changed config files, which is why we had an empty `experiments` list.